Ginger/Ginger/Environments/GingerOpsEnvWizardLib/AddGingerOpsEnvPage.xaml.cs

```diff
@@ -81,6 +81,7 @@ public void WizardEvent(WizardEventArgs WizardEventArgs)
             {
                 case EventType.Init:
                     mWizard = (AddGingerOpsEnvWizard)WizardEventArgs.Wizard;
+                    mWizard.mWizardWindow.FinishButton(false);
```

Comment:
Change name to reflect functionality of the method

Reply:
Changed to DisableFinishButton